### PR TITLE
Updating the descriptions in the default metadata file

### LINKED ISSUE #890: Update descriptions in metadata
In the metadata files, we have some descriptions like:
`"NASEM_Comp": {
      "title": "NASEM Comp data",
      "description": "Nutrients!",
      "path": "input/data/feed/NASEM_Comp_with_TDN.csv",
      "type": "csv",
      "properties": "NASEM_Comp_properties"
    },`
The description is not informative. This is just one example, we have the same problem in other fields as well. A more meaningful and informative description is needed for these.

### PR DESCRIPTION
**Pull request by ltp23**
In this PR, everyone is welcome to improve the descriptions in the `default_metadata.json` related to their work in RuFaS.

### CODE REVIEW

**Review by Isgotkowitz — COMMENTED** [review]
Made changes to Crop and Soil related descriptions. Can come back and approve when everyone is finished making their changes.

**Comment by YijingGong** [comment]
I made some changes to the metadata description. Another thing I noticed is that "cow_presynch_protocol", currently, "pattern": "^(Double OvSynch|Presynch|G6G)$". But I think it should also take 'None'/'null' because the "cow_presynch_protocol" is optional (see cow.py line 942).  But I don't know how to make the changes to the pattern. Could you help me change that? @ltp23

**Review by elle-andreen — APPROVED** [review]
All changes look appropriate to me!

File: input/metadata/default_metadata.json [hunk 1]
```diff
@@ -2,42 +2,42 @@
   "files": {
     "config": {
       "title": "Config Data",
-      "description": "Configuration Information",
+      "description": "The configuration file for the simulation.",
```
Comment (JoeWaddell) [inline input/metadata/default_metadata.json:5]: Maybe something like: "The configuration file for general simulation parameters."

(this, and all others, would apply to all metadata files)

File: input/metadata/default_metadata.json [hunk 2]
```diff
@@ -2,42 +2,42 @@
...
-      "description": "Moo Moo!",
+      "description": "The configuration file for the animal population parameters.",
       "path": "input/data/animal/animal_population.json",
       "type": "json",
       "properties": "animal_population_properties"
     },
     "crop": {
       "title": "Crop data",
-      "description": "Alfafa is fun!",
+      "description": "Contains a crop rotation specification.",
```
Comment (JoeWaddell) [inline input/metadata/default_metadata.json]: For consistency, maybe: "The configuration file for crop rotation details."

File: input/metadata/default_metadata.json [hunk 3]
```diff
@@ -121,21 +121,21 @@
...
-      "description": "Hows the feed!",
+      "description": "The configuration file for the feed quality module.",
       "path": "input/data/feed/feed_quality.csv",
       "type": "csv",
       "properties": "feed_quality_properties"
     },
     "user_feeds": {
       "title": "User Feed",
-      "description": "Bring the feed!",
+      "description": "The configuration file for the user feed module.",
```
Comment (JoeWaddell) [inline input/metadata/default_metadata.json]: The final bit should be something more like: "for user feed input."

**Review by JoeWaddell — CHANGES_REQUESTED** [review]
Looks good! I have a few suggestions that (if you agree) should be copy/pasted across metadata jsons.

**Review by JoeWaddell — APPROVED** [review]
LGTM!

**Pull request merged**